javascript-practice/src/js/controllers/dashboardController.js

```diff
@@ -46,34 +46,44 @@ export default class DashboardController {
    * The function is used to add a new user to the system.
    */
   addUser = async (
+    id,
     firstName,
     lastName,
     email,
-    phoneNumber,
+    mobile,
     role,
     userName,
     password
   ) => {
     try {
+      const emailExists = await this.service.isEmailExists(email);
+
+      if (emailExists) {
+        this.view.dashboardMessageError(`${EMAIL_EXISTS}`, 'error');
+
+        return;
+      };
+
       const newUser = new UserModel(
+        id,
         firstName,
         lastName,
         email,
-        phoneNumber,
+        mobile,
         role,
         userName,
         password
       );
       const addedUser = await this.service.addUser(newUser);
 
-      this.view.addUserMessage(`${ADD_USER_SUCCESS}`);
+      this.view.dashboardMessage(`${ADD_USER_SUCCESS}`);
       setTimeout(() => {
         this.view.closePopupUser();
         this.renderTableListUsers();
       }, 1500);
       return addedUser;
     } catch (error) {
-      this.view.addUserMessage(`${ADD_USER_FAILED}`, 'error');
+      this.view.dashboardMessageError(`${ADD_USER_FAILED}`, 'error');
```

Comment:
Please check all your codes where params are redundant. Then remove it pls

Reply:
i done fix